bin/crew

```diff
@@ -329,16 +330,21 @@ def update
     if different_version && !different_sha && has_sha
       unless @pkg.no_compile_needed?
         can_be_updated -= 1
-        puts "#{@pkg.name} has a version change but does not have updated binaries".yellow
+        updatable_packages.push(@pkg.name)
+        puts "#{@pkg.name} has a version change but does not have updated binaries".yellow unless CREW_UNATTENDED
```

Comment:
Doesn't limiting the output also limit the ability to debug?

Reply:
100%. That's why it's behind a flag!